src/app.js

```diff
@@ -74,10 +74,17 @@ export const start = args => {
     .catch(err => {
       screen.destroy();
 
-      process.stderr.write(chalk.red.bold('[ERROR]') + '\n');
-      process.stderr.write(err + '\n');
+      if (err === 'SIGTERM') {
+        process.stdout.write(chalk.white.bold('[INFO]'));
```

Comment:
process.stdout.write will print unformatted text w/o a line break at the end. If we want to include the line break, we should mimic the Error log above with the `+ '\n'` for the line break.